src/stripeSamplesView.ts

```diff
@@ -0,0 +1,26 @@
+import {StripeTreeItem} from './stripeTreeItem';
+import {StripeTreeViewDataProvider} from './stripeTreeViewDataProvider';
+import {ThemeIcon} from 'vscode';
+
+export class StripeSamplesViewProvider extends StripeTreeViewDataProvider {
+  buildTree(): Promise<StripeTreeItem[]> {
+    const items = [];
+
+    const samplesItem = new StripeTreeItem('Start with a Stripe Sample', {
+      commandString: 'createStripeSample',
+      iconPath: new ThemeIcon('repo-clone'),
+      tooltip: 'Clone a sample integration built by Stripe',
+    });
+
+    items.push(samplesItem);
+
+    const findSamplesItem = new StripeTreeItem('Find code samples', {
+      commandString: 'openSamples',
+      iconPath: new ThemeIcon('code'),
+      tooltip: 'Sample integrations built by Stripe',
+    });
+    items.push(findSamplesItem);
```

Comment:
Oh apologies if I wasn't clear! The `Find code samples` link should be available when you right-click the `Start with a Stripe Sample` link. It involves
- adding a `contextValue` to the `samplesItem`
- registering `stripe.openSamples` in the package.json similar to how `stripe.openDashboardEvent` is registered.

`stripe.openDashboardEvent` should be a good example of how to do this!

Reply:
Ah had no idea this menu existed!